services/web-app/components/mdx-notification/mdx-notification.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { ActionableNotification, Button, Column, Grid, Link } from '@carbon/react'
+import { ChevronDown, ChevronUp } from '@carbon/react/icons'
+import clsx from 'clsx'
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+
+import styles from './mdx-notification.module.scss'
+
+export const MdxNotification = ({
+  content,
+  title,
+  description,
+  href,
+  link,
+  id,
+  kind = 'error',
+  collapsible = false
+}) => {
+  const wrapperClassName = clsx({
+    'cds--inline-notification': true
+  })
+
+  const [collapsed, setIsCollapsed] = useState(false)
+
+  return (
+    <Grid narrow className={styles['error-container']}>
+      <Column sm={4} md={6} lg={16}>
+        <ActionableNotification
+          href={href}
+          className={clsx(wrapperClassName)}
+          id={id}
+          inline
+          lowContrast
+          hideCloseButton
+          kind={kind}
+        >
+          {collapsible && (
+            <Button
+              className={styles['collapse-icon']}
+              kind="ghost"
+              size="lg"
+              onClick={() => {
+                setIsCollapsed(!collapsed)
+              }}
+              hasIconOnly
+              renderIcon={collapsed ? ChevronUp : ChevronDown}
+              iconDescription={collapsed ? 'Expand' : 'Collapse'}
+              tooltipAlignment="center"
+              tooltipPosition="bottom"
+            />
+          )}
+          <Grid className={styles.grid}>
+            <Column sm={4} md={6} lg={12}>
+              <span className={styles.container}>
+                <strong className={styles.title}>{title}</strong>
+              </span>
+              <p className={styles.description}>{description}</p>
+              {!collapsed && <p className={styles.content}>{content}</p>}
+            </Column>
+            <Column sm={4} md={6} lg={4}>
+              <Link className={styles.link} href={href}>
+                {link}
+              </Link>
```

Comment:
Should this be a ghost button? 
![Screen Shot 2022-10-07 at 7 23 58 AM](https://user-images.githubusercontent.com/2753488/194552524-318d7bee-e6a9-4c1e-a00e-0f5365af5a17.png)

Reply:
whoa I. didn't even see those designs 👀

Reply:
@andreancardona was I looking at the right part of the Figma file? 😅

Reply:
@alisonjoseph yes you are :) I didn't zoom out far enough 😭 still learning Figma hehhe

Reply:
looking at this further - and I'm not sure regarding the link - I think that would just be for a component that renders successfully? But I'm not sure on what that action is calling for?

Reply:
The spec definitely has a ghost button, but something does look off.
<img width="437" alt="Screen Shot 2022-10-10 at 8 09 02 AM" src="https://user-images.githubusercontent.com/2753488/194873769-6835f37d-2d29-47fb-89cd-4d2e35d15800.png">

Looks like you're missing the 8px to the right of the ghost button?
<img width="228" alt="Screen Shot 2022-10-10 at 8 10 19 AM" src="https://user-images.githubusercontent.com/2753488/194873949-3399a2ec-11ba-4c23-983b-28874624781c.png">